notification/docs/HowToRun.md

```diff
@@ -16,44 +16,127 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Environment variables
+## Environment variable
```

Comment:
Now links to this are dead 😄 
for instance: `[how to run guide](./HowToRun.md#environment-variables)`

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/542/commits/c250367e6a21f15824b20a49509417a3d6587ebc